src/views/notifications/Histories.vue

```diff
@@ -0,0 +1,254 @@
+                              <EyeIcon class="w-3 h-3 mr-2"/>Seen
+                            </div>
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5 self-center mr-3" v-else>
+                              <EyeOffIcon class="w-3 h-3 mr-2"/>Unseen
+                            </div>
+                            <!-- END: Notification Seen -->
+                            <!-- BEGIN: Notification Type -->
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5 self-center" v-if="notification.target_id !== null">
+                              <UserIcon class="w-3 h-3 mr-2" />User Notification
+                            </div>
+                            <div class="flex items-center justify-center lg:justify-start text-gray-600 mt-5self-center" v-else>
+                              <component class="w-3 h-3 mr-2" :is="notification.icon"/>Icon Notification
+                            </div>
+                            <!-- END: Notification Type -->
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    <!-- END: Users Layout -->
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- END: Notification Creation Tab -->
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+import Header from './Components/Header'
+import axios from 'axios'
+import DefaultTab from './Components/DefaultTab'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+export default defineComponent({
+  components: {
+    DefaultTab,
+    Header
+  },
+  data() {
+    return {
+      active_tab: false,
+      history_notification_user: {},
+      users: [],
+      notifications: [],
+      permissions: {},
+      validation_error: {},
+      search: {
+        user: ''
+      }
+    }
+  },
+  mounted() {
+    this.testPagePermissions()
+    this.fetchUsers('users?paginate=0')
+  },
+  methods: {
+    testPagePermissions() {
+      axios.post('permissions/test', {
+        permissions: [
+          'notifications_update',
+          'notifications_get_single'
+        ]
+      })
+        .then((response) => {
+          this.permissions = response.data.data
+        })
+        .catch((error) => {
+          console.error(error)
+        })
+    },
+    fetchUsers(page) {
+      const loader = this.$loading.show()
+      axios.get(page)
+        .then(response => {
+          this.users = response.data
+          loader.hide()
+        })
+        .catch(error => {
+          this.validation_error = error.response.data.data.errors
+          toast.error(error.response.data.message)
+          console.error(error)
```

Comment:
Remove debug messages if not required constantly

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/7ce1e76f3284139350f2ec7415415ff6b695b571